hub/core/chunk_engine.py

```diff
@@ -472,10 +444,10 @@ def validate_num_samples_is_synchronized(self):
             )
 
 
-def _format_samples(
+def _format_read_samples(
     samples: Sequence[np.array], index: Index, aslist: bool
 ) -> Union[np.ndarray, List[np.ndarray]]:
-    """Helper function for preparing `samples` read from the chunk engine in the way the format the user expects."""
+    """Prepare samples read from chunks, like making sure the shape is as expected."""
```

Comment:
Use google style docstrings like "Prepares samples...."
Also, the docstring itself could be improved, I'm not entirely sure what this function does after reading it.